backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/Href.java

```diff
@@ -171,7 +173,7 @@ public Href buildAbsolute() {
      *
```

Comment:
please update the doc comment :-)

Reply:
Is [this](https://github.com/digitalservicebund/ris-norms/commit/b58b2661dea5bd4c31bed688918d804d6d455d62) what you meant?

Reply:
yes :)